### PR TITLE
Fix supervisor for enterprise_catalog DOS-484

### PR DESCRIPTION
**Pull request by jdmulloy**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Review by adzuci — APPROVED** [review]
:shipit:

File: playbooks/roles/supervisor/files/pre_supervisor_checks.py [hunk 1]
```diff
@@ -252,7 +263,7 @@ def edp_for_instance(instance_id):
                 subprocess.call("sudo -u supervisor ln -sf {} {}".format(available_file, link_location), shell=True)
                 report.append("Enabling service: {}".format(service))
             else:
-                raise Exception("No conf available for service: {}".format(link_location))
+                raise Exception("No conf available for service: {}".format(available_file))
```
Comment (chavesj) [inline playbooks/roles/supervisor/files/pre_supervisor_checks.py:266]: just checking - is this change intentional?

Comment (jdmulloy) [reply playbooks/roles/supervisor/files/pre_supervisor_checks.py:266]: @chavesj yes. It's a fix to the error message in the script. It's unrelated, but useful to fix because the message I got confused me. It's in a separate commit with a descriptive commit message.

**Review by chavesj — APPROVED** [review]
👍

File: playbooks/roles/enterprise_catalog/tasks/main.yml [hunk 2]
```diff
@@ -22,10 +22,10 @@
 #
 
 # write the supervisor script
-- name: "writing enterprise_catalog_workers supervisor script"
+- name: "writing enterprise_catalog_worker supervisor script"
   template:
-    src: "edx/app/supervisor/conf.d.available/enterprise-catalog-workers.conf.j2"
-    dest: "{{ supervisor_available_dir }}/enterprise-catalog-workers.conf"
+    src: "edx/app/supervisor/conf.d.available/enterprise_catalog_worker.conf.j2"
+    dest: "{{ supervisor_available_dir }}/enterprise_catalog_worker.conf"
```
Comment (adzuci) [inline playbooks/roles/enterprise_catalog/tasks/main.yml:28]: Should this be workers?
